corehq/apps/locations/static/locations/js/location_drilldown.js

```diff
@@ -190,19 +196,22 @@ hqDefine('locations/js/location_drilldown', [
             var children = [];
             if (data) {
                 children = _.sortBy(data, function(e) { return e.name; });
-
+                var accessToParent;
+                data.forEach(function(child) {
+                    accessToParent = child.have_access_to_parent;
+                });
```

Comment:
This looks like `accessToParent` will be set as the last value in the array's value of `have_access_to_parent` is that intended?

I think this would be equivalent to `accessToParent = data[-1].have_access_to_parent` (python-style. I forgot how to do negative indicies in js)